osquery/tables/system/linux/processes.cpp

```diff
@@ -163,9 +163,9 @@ void genProcessMap(const std::string& pid, QueryData& results) {
     try {
       auto offset = std::stoll(fields[2], nullptr, 16);
       r["offset"] = (offset != 0) ? BIGINT(offset) : r["start"];
-
     } catch (const std::exception& e) {
       // Value was out of range or could not be interpreted as a hex long long.
+      VLOG(1) << "Error converting value: " << e.what();
```

Comment:
This is missing some context, how do I know what to start investigating? Maybe add a reference to `offset` or `start` and the PID?